packages/go-gen/tests/cosmwasm_std__CodeInfoResponse.go

```diff
@@ -1,5 +1,5 @@
 type CodeInfoResponse struct {
-	Checksum Checksum `json:"checksum,omitempty"`
+	Checksum Checksum `json:"checksum"` // in wasmvm, this is `omitempty`
```

Comment:
I think we should remove the `omitempty` in wasmvm or an Option here. Let's figure out if we can get a guarantee this is always set.